### DISCUSSION TITLE
Two or more DC windows on macOS

### DISCUSSION THREAD

**Discussion by ghost** [doublecmd/doublecmd#1424] [Suggestions]
On macOS, sometimes it would be useful to open another DC window. ForkLift, Commander One, MaxCommander, and muCommander have that feature too. Here's how it looks in Commander One:

![New Window Commander One](https://github.com/doublecmd/doublecmd/assets/79788982/2f12e063-2ec7-4c56-aa8a-582b8625dd1b)

The user can open a new window by right-clicking the 'Commander One' icon in the dock and then selecting the entry 'New Window'.

**Comment by rich2014** [comment]
@Andreas0602 

done.

try the snapshoot please.

**Reply by okyr-ux** [reply]
Where can I get a snapshot?

**Reply by ags1234** [reply]
https://doublecmd.sourceforge.io/site/eng/snapshots.php

**Comment by ghost** [comment]
Thank you very much, @rich2014.

I've tested it, and it works fine for me – though the window handling is a bit different from other file managers. You open a new window by right-clicking the dock icon and selecting "New window", but when you select "Show all windows" afterwards, **not** all windows will be shown. That's a bit inconsistent. But it's not a big problem. I like the fact that there is an own dock icon for each window. That's even better as with other file managers. Again, thank you very much!

**Reply by rich2014** [reply]
great!

**Comment by okyr-ux** [comment]
I have Double Commander

- Version: 1.1.23 gamma
- Revision: 694
- Commit: 15e166ecd
- Build date: 2025/03/22
- Lazarus: 3.9.0.0
- FPC: 3.2.2
- Platform: x86_64-Darwin-cocoa
- OS version: Mac OS X 12.7.6

How can I open a new window?
